_delphi_utils_python/delphi_utils/flash_eval/eval_day.py

```diff
@@ -0,0 +1,332 @@
+"""Functions pertaining to running FlaSH daily."""
+
+import numpy as np
+import pandas as pd
+from scipy.stats import binom
+import boto3
+from delphi_utils.weekday import Weekday
+from .constants import HTML_LINK, STATES, BUCKET
+from .. import (
+    get_structured_logger,
+)
+
+
+def split_reporting_schedule_dfs(input_df, flash_dir, lag):
+    """Separate the input df by reporting schedule (pre-determined).
+
+    Parameters
+    ----------
+    input_df: the df to split up
+    flash_dir: the string reference directory to find all files
+    lag: difference between reporting and reference date.
+
+    Returns
+    -------
+    df of streams updated daily, less often than daily BUT has enough data to use the AR method
+    and data that's updated very infrequently and there's not enough data to use the AR method.
+    """
+    rep_sched = pd.read_csv(f'{flash_dir}/reporting_sched_{lag}.csv', index_col=0)
+    min_cut = rep_sched.loc['min_cut'][0]
```

Comment:
question: Are we assuming that `rep_sched` is sorted by `min_cut`? Or is `min_cut` the same for all rows? Should that be checked first?

Reply:
Ah, so min cut is always the last element of the series rep_sched (so the series is of all the geographies and the value is the reporting schedule, and just so that there are fewer files, I append the min cut parameter to the end). Let me know if that's unclear.

Reply:
It's part of the if statement later on in the method! [around lines 47-48]. Just to clarify, rep_sched is a series (1-d) and at the end of the 1-d series is min-cut.
e.g.

'us': 1
....
'01100':9
min_cut:7

Reply:
Thanks for explaining the format, that clears up my confusion.

> I want to note that I don't see "min_cut" used anywhere else in the flash package. Not sure if that is to-be-implemented or it lives in another package.

To clarify my comment, I meant that I don't see the logic that appends `min-cut` to the end of the `rep_sched` data (before you create the `reporting_sched` csv file).

Reply:
Ah that code is in the training functions! It's not in this commit :)